flink-runtime/src/test/java/org/apache/flink/runtime/leaderelection/LeaderElectionTest.java

```diff
@@ -210,8 +210,7 @@ public LeaderElectionService createLeaderElectionService() throws Exception {
 
         @Override
         public void setup() {
-            embeddedLeaderService =
-                    new EmbeddedLeaderService(TestingUtils.defaultExecutionContext());
+            embeddedLeaderService = new EmbeddedLeaderService(TestingUtils.defaultExecutor());
```

Comment:
Just to be sure because I had to dig a bit into it to understand what Scala's `ExecutionContext` does: We are able to do that since `EmbeddedLeaderService` requires `Executor` interface and does not use the `reportFailure` or `prepare` method that are part of Scala's `ExecutionContext`.

Reply:
Yes, and defaultExecutionContext was using the defaultExecutor underneath anyway.